src/ImageLoaders/IHex32/IntelHex32Writer.cs

```diff
@@ -0,0 +1,215 @@
+﻿#region License
+/* 
+ * Copyright (C) 2017-2018 Christian Hostelet.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Reko.ImageLoaders.IntelHex32
+{
+
+    /// <summary>
+    /// A writer capable of writing an Intel Hexadecimal 32-bit object format stream.
+    /// </summary>
+    public class IntelHex32Writer : IDisposable
```

Comment:
`IntelHex32Writer` and `IntelHex32AddressType` classes are unused. Shouldn't be removed. Or maybe you are going to use them in future